### DISCUSSION TITLE
Can I change Farmer Key and Pool Contract Address of existing plot files?

### DISCUSSION THREAD

**Discussion by brightways021** [Chia-Network/chia-blockchain#8164] [General]
Hi. Can I change Farmer Key and Pool Contract Address of existing plot files? If yes, please teach.tnx

**Comment by emlowe** [comment]
no

**Comment by BrandtH22** [comment]
@brightways021 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
